src/Core/Content/MeasurementSystem/Service/MeasurementUnitConverterInterface.php

```diff
@@ -0,0 +1,11 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Content\MeasurementSystem\Service;
+
+use Shopware\Core\Framework\Log\Package;
+
+#[Package('inventory')]
+interface MeasurementUnitConverterInterface
```

Comment:
is there a reason this is an interface? Do we want to allow decoration here? Then we should use an abstract class with getDecorated right? That way we can at least add new methods to the interface without breaking change, 

Same for the provider interface